Good-Bids/goodbids
# [#371, #373, #369] Stripe Invoices

# Summary

This PR handles the generation and storage of Stripe Invoices. This will occur immediately after the WordPress Invoice is created.

## Issues

* #371 
* #373 
* #369 

## Testing Instructions

1. Vist a Nonprofit site
2. Wait for an Auction to end - Be sure there are bids and there is an Auction total.
3. Once it ends, an Invoice should be automatically generated and visible on the WP Admin > Invoices screen for that Nonprofit.
4. If it isn't automatically generated (due to cron not firing), you should be able to visit the Auction on the front-end of the site and that should trigger the "close" event which generates the invoice.
5. You should notice an Invoice ID (hovering over it will show the full ID)
6. The Pay Now button should be active and take you to the Invoice.

## Screenshots

<img width="1094" alt="Screenshot 2024-02-07 at 11 05 06 AM" src="https://github.com/Good-Bids/goodbids/assets/122309362/eb6ca8d0-4866-403d-b9e4-0a9733d65f66">
